docs/tutorials/orc.ipynb

```diff
@@ -0,0 +1,398 @@
+{
```

Comment:
<p>This can be a new section titled "Build, compile and train the model"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/io/pull/1465/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>